settings.gradle

```diff
@@ -29,3 +29,8 @@ gradle.ext.scalafmt = [
     version: '1.5.0',
     config: new File(rootProject.projectDir, '.scalafmt.conf')
 ]
+
+gradle.ext.scoverage = [
+        deps: ['org.scoverage:scalac-scoverage-plugin_2.11:1.3.1',
+               'org.scoverage:scalac-scoverage-runtime_2.11:1.3.1']
+]
```

Comment:
Please fix indentation of all lists, they all seem to be double-indented whereas the "old" code uses single-identation.

Reply:
Done now